samples/nextjs/src/lib/sitemap-fetcher.ts

```diff
@@ -21,10 +21,25 @@ export class SitecoreSitemapFetcher {
     });
 
     this._disconnectedSitemapService = new DisconnectedSitemapService(
-      (manifest as unknown) as ManifestInstance
+      (this.getManifest() as unknown) as ManifestInstance
     );
   }
 
+  /**
+   * Get sitecore-import.json manifest
+   */
+  private getManifest() {
+    if (process.env.JSS_MODE !== 'disconnected') return null;
+
+    try {
+      const manifest = require('sitecore/manifest/sitecore-import.json');
+
+      return manifest;
+    } catch (error) {
+      throw Error("[Disconnected Export] sitecore-import.json doesn't exist. Generate it first");
```

Comment:
Maybe we should be more specific on how they generate it. e.g. "Please make sure you've started the disconnected proxy `npm run start:disconnected-proxy`"